roles/ceph-rgw/defaults/main.yml

```diff
@@ -21,6 +21,28 @@ cephx: true
 rgw_pull_port: "{{ radosgw_civetweb_port }}"
 rgw_pull_proto: "http"
 
+########
+#TUNING#
+########
+# The below values were determined to be safe defaults to 
+# prevent users from putting too many objects into their
+# buckets which causes performance and stability issues.
+#ceph_conf_overrides:
```

Comment:
No keep it here :). However, this won't work as expected I'm afraid since we have one rgw section per rgw. The issue reported into this PR: https://github.com/ceph/ceph-ansible/pull/1018 should be fixed before we can merge this.